### PR TITLE
Infra 3637 - Add Owl and Lightbox2

### PR DESCRIPTION
**Pull request by ericpoirier**
Change-Id: Ifaaed101d90b1f6540526b7c437722ada80c2d88
Signed-off-by: Eric Poirier <eric.poirier@eclipse-foundation.org>

### CODE REVIEW

File: less/quicksilver/europe2020/drupal.less [hunk 1]
```diff
@@ -19,6 +19,11 @@
 @import "_variables.less";
 @import "template.less";
 
+@import "../../../../owl.carousel/dist/assets/owl.carousel.min.css";
```
Comment (chrisguindon) [inline less/quicksilver/europe2020/drupal.less]: Please use @import '~/owl.carousel...

Comment (ericpoirier) [reply less/quicksilver/europe2020/drupal.less]: Done

Comment (ericpoirier) [reply less/quicksilver/europe2020/drupal.less]: After some testing this was the correct path to use: ~[path]

Instead of: ~/[path]

Comment (chrisguindon) [reply less/quicksilver/europe2020/drupal.less]: Correct!

**Pull request merged**